scripts/dev/replace-new-version.sh

```
@@ -0,0 +1,41 @@
+# 현재 시간으로 로그 파일명 생성
+LOG_FILE="$LOG_DIR/mykku-$(date +%Y%m%d_%H%M%S).log"
+
+echo "Starting application with log file: $LOG_FILE"
+
+sudo nohup java \
+    -Dspring.profiles.active=dev \
+    -Duser.timezone=Asia/Seoul \
+    -Dserver.port=8080 \
+    -Ddd.service=debate-timer \
+    -Ddd.env=dev \
+    -jar "$JAR_FILE" > "$LOG_FILE" 2>&1 &
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)